ApollosProject/apollos-prototype
# Fixes padding issue with close icon when image is not available

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR gives logic to the padded view wrapping the content item to receive vertical padding when an image is absent. 

### What design trade-offs/decisions were made?

### How do I test this PR?

Run the app to confirm the X icon is not covering the title of a imageless content item. 

### What automated tests did you write?

n.a.

## TODO:

- [x] Closes [#1174]

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
